### PR TITLE
[hevce] Enabled LowPower mode on CNL

### PR DESCRIPTION
**Pull request by DenWolf**
Signed-off-by: Denis Volkov <denis.volkov@intel.com>

### CODE REVIEW

File: _studio/hevce_hw/h265/include/mfx_h265_encode_vaapi.h [hunk 1]
```diff
@@ -247,7 +247,7 @@ mfxStatus SetSkipFrame(
         virtual
         VAEntrypoint GetVAEntryPoint()
         {
-            return VAEntrypointEncSlice;
+            return IsOn(m_videoParam.mfx.LowPower) ? VAEntrypointEncSliceLP : VAEntrypointEncSlice;
```
Comment (onabiull) [inline _studio/hevce_hw/h265/include/mfx_h265_encode_vaapi.h]: Did your check that this correctly handled on pre-CNL platforms?

**Comment by DenWolf** [comment]
@onabiull , I will update this patch today

**Comment by onabiull** [comment]
My original question was related to runtime behavior with API 1.27 or higher. Does HEVC Encoder return correct MFX_ERR_UNSUPPORTED on pre-CNL platforms with this change?

**Comment by DenWolf** [comment]
This patch doesn't change behavior of this function on pre-CNL platforms - in this case GetVAEntryPoint() returns  VAEntrypointEncSlice as previously. Or I miss something?
So, if we cannot get MFX_ERR_UNSUPPORTED on pre-CNL platforms for HEVC VDEnc, I think this is an incorrect behavior and it need to be fixed by another patches.

**Pull request merged**